src/logic/useReading.tsx

```diff
@@ -26,27 +26,27 @@ export const useReading = () => {
 // Provider hook that creates auth object and handles state
 function useProvideReading() {
   const [reading, setReading] = useState(InitialReading);
-  // const dummyGenerator = DummyDataGenerator(setReading, 20);
+  const dummyGenerator = DummyDataGenerator(setReading, 20);
   const serialDataHandler = SerialDataHandler({ baudRate: 115200 }, setReading);
 
   // Subscribe to user on mount
   // Because this sets state in the callback it will cause any ...
   // ... component that utilizes this hook to re-render with the ...
   // ... latest auth object.
   useEffect(() => {
-    serialDataHandler.startUsbListener();
```

Comment:
keep this change locally, you shouldn't commit this change in for now as we use it for testing purposes

Reply:
sorry forgot to revert it back, this is in the last commit